@zicklam 
I'm seeing the following unit test failures, in `awx/main/tests/functional/test_notifications.py`


```
_________________________ test_basic_parameterization __________________________
Traceback (most recent call last):
  File "/awx_devel/awx/main/tests/functional/test_notifications.py", line 34, in test_basic_parameterization
    assert response.status_code == 201
AssertionError: assert 400 == 201
 +  where 400 = <Response status_code=400, "application/json">.status_code
----------------------------- Captured stderr call -----------------------------
2019-05-06 15:28:37,556 WARNING  awx.api.generics status 400 received by user admin-poster attempting to access /api/v2/notification_templates/ from 127.0.0.1
------------------------------ Captured log call -------------------------------
generics.py                221 WARNING  status 400 received by user admin-poster attempting to access /api/v2/notification_templates/ from 127.0.0.1
____________________ test_inherited_notification_templates _____________________
Traceback (most recent call last):
  File "/awx_devel/awx/main/tests/functional/test_notifications.py", line 87, in test_inherited_notification_templates
    assert response.status_code == 201
AssertionError: assert 400 == 201
 +  where 400 = <Response status_code=400, "application/json">.status_code
----------------------------- Captured stderr call -----------------------------
2019-05-06 15:28:38,011 WARNING  awx.api.generics status 400 received by user admin-poster attempting to access /api/v2/notification_templates/ from 127.0.0.1
------------------------------ Captured log call -------------------------------
generics.py                221 WARNING  status 400 received by user admin-poster attempting to access /api/v2/notification_templates/ from 127.0.0.1
______________________ test_custom_environment_injection _______________________
Traceback (most recent call last):
  File "/awx_devel/awx/main/tests/functional/test_notifications.py", line 149, in test_custom_environment_injection
    assert response.status_code == 201
AssertionError: assert 400 == 201
 +  where 400 = <Response status_code=400, "application/json">.status_code
----------------------------- Captured stderr call -----------------------------
```

For the first test, it looks like the notification template detail endpoint is expecting `webhook_no_verify_ssl` as a required variable, which isn't being provided in the test:

```
{'__all__': ["Missing required fields for Notification Configuration: ['webhook_no_verify_ssl']."]}
```

Is the intent here to _not_ have a default value for this field? As long as the other notification types that have this field (rocket chat and mattermost, for example) behave the same way then I think that is fine - To resolve these you'll need to double-check that the behavior is consistent with the other notification types and then update the tests to provide values for the new field.

--- a/awx/main/notifications/webhook_backend.py
+++ b/awx/main/notifications/webhook_backend.py
@@ -15,7 +15,7 @@ logger = logging.getLogger('awx.main.notifications.webhook_backend')
 class WebhookBackend(AWXBaseEmailBackend):
 
     init_parameters = {"url": {"label": "Target URL", "type": "string"},
-                       "disable_ssl_verification": {"label": "Verify SSL", "type": "bool"},
+                       "disable_ssl_verification": {"label": "Verify SSL", "type": "bool", "default": false},
                        "headers": {"label": "HTTP Headers", "type": "object"}}
     recipient_parameter = "url"
     sender_parameter = None

--- a/awx/main/notifications/webhook_backend.py
+++ b/awx/main/notifications/webhook_backend.py
@@ -15,7 +15,7 @@ logger = logging.getLogger('awx.main.notifications.webhook_backend')
 class WebhookBackend(AWXBaseEmailBackend):
 
     init_parameters = {"url": {"label": "Target URL", "type": "string"},
-                       "disable_ssl_verification": {"label": "Verify SSL", "type": "bool", "default": false},
+                       "disable_ssl_verification": {"label": "Verify SSL", "type": "bool", "default": False},
                        "headers": {"label": "HTTP Headers", "type": "object"}}
     recipient_parameter = "url"
     sender_parameter = None

--- a/awx/main/notifications/webhook_backend.py
+++ b/awx/main/notifications/webhook_backend.py
@@ -15,13 +15,13 @@ logger = logging.getLogger('awx.main.notifications.webhook_backend')
 class WebhookBackend(AWXBaseEmailBackend):
 
     init_parameters = {"url": {"label": "Target URL", "type": "string"},
-                       "webhook_no_verify_ssl": {"label": "Verify SSL", "type": "bool"},
+                       "disable_ssl_verification": {"label": "Verify SSL", "type": "bool"},
                        "headers": {"label": "HTTP Headers", "type": "object"}}
     recipient_parameter = "url"
     sender_parameter = None
 
-    def __init__(self, headers, webhook_no_verify_ssl=False, fail_silently=False, **kwargs):
-        self.webhook_no_verify_ssl = webhook_no_verify_ssl
+    def __init__(self, headers, disable_ssl_verification=False, fail_silently=False, **kwargs):
+        self.disable_ssl_verification = disable_ssl_verification
         self.headers = headers
         super(WebhookBackend, self).__init__(fail_silently=fail_silently)
 
@@ -36,7 +36,7 @@ class WebhookBackend(AWXBaseEmailBackend):
             r = requests.post("{}".format(m.recipients()[0]),
                               json=m.body,
                               headers=self.headers,
-                              verify=(not self.webhook_no_verify_ssl))
+                              verify=(not self.disable_ssl_verification))
             if r.status_code >= 400:
                 logger.error(smart_text(_("Error sending notification webhook: {}").format(r.text)))
                 if not self.fail_silently:

--- a/awx/main/tests/functional/test_notifications.py
+++ b/awx/main/tests/functional/test_notifications.py
@@ -28,7 +28,7 @@ def test_basic_parameterization(get, post, user, organization):
                          description="test webhook",
                          organization=organization.id,
                          notification_type="webhook",
-                         notification_configuration=dict(url="http://localhost",
+                         notification_configuration=dict(url="http://localhost", disable_ssl_verification=False,
                                                          headers={"Test": "Header"})),
                     u)
     assert response.status_code == 201
@@ -81,7 +81,7 @@ def test_inherited_notification_templates(get, post, user, organization, project
                              description="test webhook {}".format(nfiers),
                              organization=organization.id,
                              notification_type="webhook",
-                             notification_configuration=dict(url="http://localhost",
+                             notification_configuration=dict(url="http://localhost", disable_ssl_verification=False,
                                                              headers={"Test": "Header"})),
                         u)
         assert response.status_code == 201
@@ -143,7 +143,7 @@ def test_custom_environment_injection(post, user, organization):
                          description="test webhook",
                          organization=organization.id,
                          notification_type="webhook",
-                         notification_configuration=dict(url="https://example.org",
+                         notification_configuration=dict(url="https://example.org", disable_ssl_verification=False,
                                                          headers={"Test": "Header"})),
                     u)
     assert response.status_code == 201

--- a/awx/ui/client/src/notifications/notificationTemplates.form.js
+++ b/awx/ui/client/src/notifications/notificationTemplates.form.js
@@ -399,7 +399,7 @@ export default ['i18n', function(i18n) {
                 subForm: 'typeSubForm',
                 ngDisabled: '!(notification_template.summary_fields.user_capabilities.edit || canAdd)'
             },
-            webhook_no_verify_ssl: {
+            disable_ssl_verification: {
                 label: i18n._('Disable SSL Verification'),
                 type: 'checkbox',
                 ngShow: "notification_type.value == 'webhook' ",

